lib/provider.js

```diff
@@ -253,7 +252,11 @@ const parseOutput = (text) => {
   text = text && text.slice(0, text.indexOf('\n'));
 
   if (!text) {
-    throw noOutputFromOracleError();
+    atom.notifications.addWarning('Elm Autocomplete Error', {
```

Comment:
Oh wacky, hadn't thought of that case! The filepath exists before the file is saved!?

![keanu-woah](https://cloud.githubusercontent.com/assets/126604/12064179/d2770cec-afbc-11e5-9afe-95abce252f0a.png)

In that case, let's create a `fileExists` function that:

  1. Ensures the filepath looks legit
  2. Checks to see if the file actually exists (using node's fs#statsSync).

Then we can use that in the `shouldPreventAutoComplete` function. How does that sound?